libs/atqr/domain/src/lib/domain-spec/aggegrates/Challenge Aggregate.disl

```diff
@@ -21,7 +21,7 @@ domain ATQR
 
         ⚙️ recipe:
           @goalId: GoalId
-          @owner: User
+          @owner: Contact
```

Comment:
owner is of type "User"

Reply:
No, I asked saulo and he told me that the right type for owner was Contact 
I asked him since that were two owner props , one with User  and the other with Contact types. He specified that it was supose to be Contact

Reply:
ok!